src/OpenTracing/SpanBuilder.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace OpenTracing
+{
+    /// <summary>
+    /// Helper builder for convenience to construct startoptions and create a new span.
+    /// </summary>
+    public class SpanBuilder
```

Comment:
this is an interface in Java and I think that's the right thing for the core layer... esp for the noop impl.